### PR TITLE
Render images instead of circles for data points in LineChart

### PR DESCRIPTION
**Pull request by CHOIMINSEOK**
### Changes
**Implement Image instead of circles for data points in LineCharts**

### How to use
1. Pass resolved Image, width size, height size on LineData Props
2. Should resolve image through Image.resolveAssetSource()

```javascript
// refer to https://stackoverflow.com/a/50894993/12330603
function convertImageToImageBundleForNativeModule(image: any): any {
  return Image.resolveAssetSource(image);
}
 ```

### updated interface for LineData:
```javascript 
const lineData = PropTypes.shape({
  dataSets: PropTypes.arrayOf(
    PropTypes.shape({
      values: PropTypes.arrayOf(
        PropTypes.oneOfType([
          PropTypes.shape({
            x: PropTypes.number,
            y: PropTypes.number.isRequired,
            marker: PropTypes.string,
           // props for image
            icon: PropTypes.shape({
              bundle: PropTypes.object,
              width: PropTypes.number,
              height: PropTypes.number
            })
          //////
          }),
          PropTypes.number
        ])
      ),
      label: PropTypes.string.isRequired,
...
```

### Screens
| Android  | iOS |
| ------------- | ------------- |
| image |  image |

### Related Issue
 #371

### CODE REVIEW

File: ios/ReactNativeCharts/line/LineDataExtract.swift [hunk 1]
```diff
@@ -93,17 +94,41 @@ class LineDataExtract : DataExtract {
...
             }
             
+            if dict["icon"].exists() {
+                let icon = dict["icon"]
+                if icon["bundle"].dictionary != nil {
+                    let bundle = icon["bundle"];
+                    
+                    let uiImage = RCTConvert.uiImage(bundle.dictionaryObject);
+                    let width = CGFloat(icon["width"].numberValue)/4;
+                    let height = CGFloat(icon["height"].numberValue)/4;
```
Comment (wuxudong) [inline ios/ReactNativeCharts/line/LineDataExtract.swift]: what is the meaning of `4`?

**Comment by angelos3lex** [comment]
@wuxudong Do you plan on a new release? This is added on master a long time ago, but it's not yet released

**Comment by taxidermic** [comment]
Hi! Could you provide an example, please? Couldn't get it work :(


```
import icon from './icon.png'
...

data: {
          dataSets: [{
            values: [{
              x: 4,
              y: 110,
              marker: 'test',
              icon: Image.resolveAssetSource(icon),
            }, {
              x: 5,
              y: 110,
            }, {
              x: 6,
              y: 105,
              icon: Image.resolveAssetSource(icon),
            }, {
              x: 7,
              y: 113,
              icon: Image.resolveAssetSource(icon),
            }],
            label: 'C',
          }],
        },
...
```

**Comment by wuxudong** [comment]
@taxidermic Pay attention to the schema
```
icon: PropTypes.shape({
  bundle: PropTypes.object,
  width: PropTypes.number,
  height: PropTypes.number
}
```

**Comment by anniewey** [comment]
@CHOIMINSEOK thanks for this feature, however im getting blurry images when passing the `icon` to the chart on both ios and android. did u know any fix on this?

**Pull request merged**